src/BloomExe/CollectionTab/LibraryListView.cs

```diff
@@ -92,19 +91,48 @@ enum ButtonManagementStage
 			_showHistoryMenu.Visible = _showNotesMenu.Visible = Settings.Default.ShowSendReceive;
 
 			if(Settings.Default.ShowExperimentalCommands)
-				_settingsProtectionHelper.ManageComponent(_exportToXMLForInDesignToolStripMenuItem);//we are restriting it because it opens a folder from which the user could do damage
+				_settingsProtectionHelper.ManageComponent(_exportToXMLForInDesignToolStripMenuItem);//we are restricting it because it opens a folder from which the user could do damage
 			_exportToXMLForInDesignToolStripMenuItem.Visible = Settings.Default.ShowExperimentalCommands;
 
 			SetupBookDropdownIcon();
 			_bookContextMenu.Closed += _bookContextMenu_Closed;
+			_bookContextMenu.Opening += _bookContextMenu_Opening;
+		}
+
+		// adjust the context menu item visibility based on what sort of collection we're in
+		void _bookContextMenu_Opening(object sender, CancelEventArgs e)
+		{
+			var btn = (sender as ContextMenuStrip).SourceControl as Button;
```

Comment:
The menu items are mostly created in the designer... so not really a good place to put extra comments, as they might get overwritten later. :(